front/src/common/api/osrdEditoastApi.ts

```diff
@@ -1699,6 +1711,46 @@ export type GetV2TimetableByIdConflictsApiArg = {
   id: number;
   infraId: number;
 };
+export type PostV2TimetableByIdStdcmApiResponse =
+  /** status 201 The simulation result */
+  | {
+      departure_time: string;
+      path: PathfindingResultSuccess;
+      simulation: SimulationResponse;
+      status: 'success';
+    }
+  | {
+      status: 'path_not_found';
+    };
+export type PostV2TimetableByIdStdcmApiArg = {
+  /** The infra id */
+  infra: number;
+  /** timetable_id */
+  id: number;
+  body: {
+    comfort: Comfort;
+    margin?: MarginValue | null;
```

Comment:
What margins value does it refer to ?

Reply:
It refers to this structure : https://github.com/OpenRailAssociation/osrd/blob/0393270dbee6341fd747a3face2feef542a1731c/front/src/common/api/osrdEditoastApi.ts#L3320

Reply:
That's the unit value, but where is the value of this unit ? Where, in the new interface, can we chose a margin unit and its value ?

Reply:
My bad, I didn't read the type properly